plugins/download.py

```diff
@@ -122,6 +129,34 @@ def _download_source(self, pkg_specs):
         locations = sorted([pkg.localPkg() for pkg in pkgs])
         return locations
 
+    def _download_debuginfo(self, pkg_specs):
+        """Download debuginfo packages to dnf cache."""
+        dbg_pkgs = set()
+        q = self.base.sack.query()
+        q = q.available()
+
+        for pkg in self._get_packages(pkg_specs):
+            for dbg_name in [dnfpluginscore.lib.package_debug_name(pkg),
+                            dnfpluginscore.lib.package_source_debug_name(pkg)]:
+                dbg_available = q.filter(
+                                        name=dbg_name,
+                                        epoch=int(pkg.epoch),
+                                        version=pkg.version,
+                                        release=pkg.release,
+                                        arch=pkg.arch
+                                    )
+                dbg_found = False
+                for p in dbg_available:
+                    dbg_pkgs.add(p)
+                    dbg_found = True
+
+                if dbg_found:
```

Comment:
It is used to break out of the for dbg_name in loop. I can give it more testing once #113 and its corresponding test changes are merged.